MapleServer2/Network/Session.cs

```diff
@@ -79,6 +79,13 @@ public void Init([NotNull] TcpClient client)
         RecvCipher = new(VERSION, Riv, BLOCK_IV);
     }
 
+    public bool IsLocalHost()
+    {
+        string[] ipInfo = Name.Split(":");
+
+        return ipInfo?[0] == "127.0.0.1";
```

Comment:
Maybe turn this string into a constant that you can then refer in all the other places?